From: Simon McVittie <smcv@debian.org>
Date: Fri, 17 Aug 2018 08:55:33 +0100
Subject: Remove version from polkit-gobject dependency

It was versioned like this to require the polkit ITS rules, but in
Debian we backported those into 0.105.

Forwarded: not-needed, Debian-specific
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index eb5989b..fdf5992 100644
--- a/meson.build
+++ b/meson.build
@@ -306,7 +306,7 @@ privileged_group = get_option('privileged_group')
 enable_admin = get_option('admin')
 if enable_admin
   libcap_dep = dependency('libcap')
-  polkit_gobject_dep = dependency('polkit-gobject-1', version: '>= 0.114')
+  polkit_gobject_dep = dependency('polkit-gobject-1')
 endif
 
 # *** Check if we should build with http backend ***
